core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/presto/PrestoDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(PrestoDatabaseConnection databaseConnection) {
     public Class<PrestoDatabaseConnection> appliesTo() {
         return PrestoDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(PrestoDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class